.github/workflows/rtw-env/rtw-env.lock

```diff
@@ -0,0 +1,179 @@
+https://conda.anaconda.org/conda-forge/noarch/aniso8601-7.0.0-py_0.tar.bz2
+https://conda.anaconda.org/conda-forge/noarch/async-timeout-4.0.3-pyhd8ed1ab_0.conda
+https://conda.anaconda.org/conda-forge/linux-64/cryptography-42.0.7-py311h4a61cc7_0.conda
+https://conda.anaconda.org/conda-forge/noarch/graphql-core-2.2-py_0.tar.bz2
+https://conda.anaconda.org/conda-forge/linux-64/harfbuzz-8.5.0-hfac3d4d_0.conda
+https://conda.anaconda.org/conda-forge/noarch/importlib_metadata-7.1.0-hd8ed1ab_0.conda
+https://conda.anaconda.org/conda-forge/linux-64/numpy-1.26.4-py311h64a7726_0.conda
+https://conda.anaconda.org/conda-forge/noarch/requests-2.31.0-pyhd8ed1ab_0.conda
+https://conda.anaconda.org/conda-forge/linux-64/urwid-2.6.12-py311h331c9d8_0.conda
+https://conda.anaconda.org/conda-forge/linux-64/contourpy-1.2.1-py311h9547e67_0.conda
+https://conda.anaconda.org/conda-forge/noarch/graphql-relay-2.0.1-py_0.tar.bz2
+https://conda.anaconda.org/conda-forge/linux-64/pandas-1.5.3-py311h2872171_1.conda
+https://conda.anaconda.org/conda-forge/linux-64/pango-1.52.2-ha41ecd1_0.conda
+https://conda.anaconda.org/conda-forge/linux-64/secretstorage-3.3.3-py311h38be061_2.conda
+https://conda.anaconda.org/conda-forge/noarch/graphene-2.1.9-pyhd8ed1ab_0.tar.bz2
+https://conda.anaconda.org/conda-forge/linux-64/gtk2-2.24.33-h280cfa0_4.conda
+https://conda.anaconda.org/conda-forge/linux-64/keyring-23.13.1-py311h38be061_0.conda
+https://conda.anaconda.org/conda-forge/linux-64/librsvg-2.58.0-hadf69e7_1.conda
+https://conda.anaconda.org/conda-forge/linux-64/matplotlib-base-3.8.4-py311h54ef318_0.conda
+https://conda.anaconda.org/conda-forge/noarch/cylc-flow-base-8.2.7-pyh707e725_0.conda
+https://conda.anaconda.org/conda-forge/linux-64/graphviz-9.0.0-h78e8752_1.conda
+https://conda.anaconda.org/conda-forge/noarch/metomi-rose-base-2.2.0-pyh707e725_0.conda
+https://conda.anaconda.org/conda-forge/noarch/cylc-flow-8.2.7-pyh48b6436_0.conda
+https://conda.anaconda.org/conda-forge/noarch/cylc-rose-1.3.4-pyh707e725_0.conda
+https://conda.anaconda.org/conda-forge/linux-64/pygraphviz-1.12-py311hbf5cbc9_0.conda
+https://conda.anaconda.org/conda-forge/noarch/metomi-rose-2.2.0-pyh707e725_0.conda
+https://conda.anaconda.org/conda-forge/noarch/sphinxcontrib-applehelp-1.0.8-pyhd8ed1ab_0.conda
+https://conda.anaconda.org/conda-forge/noarch/sphinxcontrib-devhelp-1.0.6-pyhd8ed1ab_0.conda
+https://conda.anaconda.org/conda-forge/noarch/sphinxcontrib-htmlhelp-2.0.5-pyhd8ed1ab_0.conda
+https://conda.anaconda.org/conda-forge/noarch/sphinxcontrib-jquery-4.1-pyhd8ed1ab_0.conda
+https://conda.anaconda.org/conda-forge/noarch/sphinx_rtd_theme-2.0.0-pyha770c72_0.conda
+https://conda.anaconda.org/conda-forge/noarch/sphinxcontrib-qthelp-1.0.7-pyhd8ed1ab_0.conda
+https://conda.anaconda.org/conda-forge/noarch/sphinx-7.3.7-pyhd8ed1ab_0.conda
+https://conda.anaconda.org/conda-forge/noarch/sphinxcontrib-serializinghtml-1.1.10-pyhd8ed1ab_0.conda
```

Comment:
just putting a placeholder here to remind myself about the need to get this out :+1:

Reply:
This file no longer exists in the `recipe_test_workflow_prototype` branch 👍

Reply:
@valeriupredoi, would you be happy to resolve this conversation now #3816 has been merged? 🤞 😊